pkg/segment/query/iqr/intermediateQueryResult_test.go

```diff
@@ -505,6 +505,21 @@ func Test_DiscardAfter(t *testing.T) {
 	assert.Equal(t, 1, iqr.NumberOfRecords())
 }
 
+func getKnownValues() map[string][]utils.CValueEnclosure {
```

Comment:
nit: rename to indicate this is only for testing

Reply:
addressed in the latest commit.